HHS/Head-Start-TTADP
# [TTAHUB-689] Supporting Attachments Redesign

## Description of change
Supporting Attachments Redesign


## How to test

The 'Supporting Attachments' page should match the mockup in TTAHUB-689.

Some of the more notable changes include:
- The page now appears after the goals page.
- Topics and resources have been removed.
- Wording on the upload text and button have changed.
- Buttons Save Draft, Save and Continue, and Back are in a new order (on all pages).

We also need to make sure the user can upload and remove attachments. Once a report has entered review the attachments should show accordingly.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-689


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
